examples/cxx/request_response/src/client.cpp

```diff
@@ -49,7 +50,7 @@ auto main() -> int {
         request_counter += 1;
         // send all other requests by using zero copy API
         auto request = client.loan_uninit().expect("loan successful");
-        auto initialized_request = request.write_payload(request_counter);
+        auto initialized_request = request.write_payload(std::move(request_counter));
```

Comment:
Okay, it seems with `auto request_counter = 0U;` this change can be reverted and there is no conversion warning. Can you use `0U` for `request_counter` and `response_counter`